tests/unit/test_variants.py

```diff
@@ -70,3 +73,112 @@ def testPhasesetZero(self):
 
     def testGenotypeHaploid(self):
         self.verifyGenotypeConversion("1", "376", [1], None)
+
+
+class TestAbstractVariantSet(unittest.TestCase):
+    """
+    Unit tests for the abstract variant set.
+    """
+    def setUp(self):
+        self._vsname = "testVariantSet"
```

Comment:
We tend to avoid abbreviations (although it can be tedious, I agree). Prefer ``self._variantSetName`` here.